### DISCUSSION TITLE
spectrum similarity values

### DISCUSSION THREAD

**Discussion by amrelguo** [vdemichev/DiaNN#729] [Q&A]
Hi Vadim,
I want to filter the identifications i got, considering spectrum similarity values. however, I am wondering how this spectrum similarity calculated and which parameters taken into consideration when it was calculated because when i plotted the DDA spectrum (from DDA spectral library)against DIA spectrum (from lib-report.tsv output file or XIC) for certain peptide, I found that no correlation between the values received from spectrum similarity column and the similarity expereinceed from plotting the data in terms of fragment m/z and intensity. does the fragment ions excluded from asssay in lib-report.tsv file(ExcludeFromAssay  = TRUE) not considered in spectrum similarity calculation)?

Best, 
Amr

**Comment by vdemichev** [comment]
Hi Amr,

This is an experimental quality score, it's not well tested or documented, so I don't suggest using it.
It's more useful for internal workings of DIA-NN, for identification purposes, not for post-filtering. The latter is better done using other scores reported by DIA-NN, depending on the purpose of the filtering.

Best,
Vadim

**Reply by amrelguo** [reply]
I got it. thanks so much for your reply.
